docs/contributors/contributors-guide.md

```diff
@@ -145,7 +149,9 @@ The "Contributors" tab allows you to view more detailed information about other
 ![contributors insight list](../../static/img/contributors-insights.png)
 
 :::tip
+
 Want to learn more about your peers? You can select them and add them to a list.
+
 :::
 
 To learn more about these features, see [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).
```

Comment:
As we're talking about how to read and understand the data, we might be able to clarifiy this with wording adjustment as below:

```suggestion
To learn and understand more about the data provided, see [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).
```

Any other idea on the wording?

If we agreed on the wording, we might as well will reword the same sentence in the maintainer guide.

Reply:
How about this way @adiati98? It’ll clarify to current and potential users to expand their knowledge a bit more.

```suggestion
Want to learn other ways to use these features? Check out [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).
```

Reply:
In my understanding, what we want to say here is for folks to learn about how to read and understand the data provided in each tab. And these are not part of features.

Would love to hear @BekahHW opnion about this.

Reply:
@adiati98's correct here.